### PR TITLE
Add Grid flex fill rest demo

### PR DESCRIPTION
**Pull request by zombieJ**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

Missing demo for flex Grid fill rest demo.

### 💡 Solution

Add one demo:

### 📝 Changelog description

None.

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/grid/demo/flex.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ ReactDOM.render(
       <Col span={4}>col-4</Col>
       <Col span={4}>col-4</Col>
     </Row>
+
+    <p>sub-element fill rest</p>
```
Comment (afc163) [inline components/grid/demo/flex.md]: 单独放一个 demo 吧，可以描述得更清晰一点。

File: components/grid/demo/flex.md [hunk 2]
```diff
@@ -59,6 +59,13 @@ ReactDOM.render(
...
+    <Row type="flex">
+      <Col style={{ width: 100 }}>100px</Col>
+      <Col span={4}>col-4</Col>
+      <Col style={{ flex: 'auto' }}>fill rest</Col>
```
Comment (afc163) [inline components/grid/demo/flex.md]: `style={{ flex: 'auto' }}` 是不是不需要？

Comment (zombieJ) [reply components/grid/demo/flex.md]: 需要的，我们默认没加 flex: auto：

Comment (afc163) [reply components/grid/demo/flex.md]: 感觉可以默认设一下。

Comment (zombieJ) [reply components/grid/demo/flex.md]: 默认会 break 这种场景：
```jsx
<Row type="flex">
  <Col style={{ width: 200, background: 'green' }}>left</Col> // 或者是 className
  <Col style={{ background: 'red' }}>right</Col>
</Row>
```

File: components/grid/demo/flex-fill-rest.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      <Col style={{ flex: 'auto' }}>fill rest</Col>
+      <Col style={{ width: 100 }}>100px</Col>
+      <Col style={{ flex: 'auto' }}>fill rest</Col>
+    </Row>
+
+    <p>Merge with span</p>
+    <Row type="flex">
+      <Col style={{ width: 100 }}>100px</Col>
+      <Col span={4}>col-4</Col>
+      <Col style={{ flex: 'auto' }}>fill rest</Col>
```
Comment (afc163) [inline components/grid/demo/flex-fill-rest.md]: 直接给 Row[type="flex"] 的 children 加上 `flex: auto` 吧。

Comment (zombieJ) [reply components/grid/demo/flex-fill-rest.md]: 看一下我上面的例子，如果用户用了 `className` or `style` 就出问题了。

File: components/grid/demo/flex-fill-rest.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  en-US: Flex Fill Rest
+---
+
+## zh-CN
+
+Flex 填充剩余空间。
+
+## en-US
+
+Flex fill rest space.
```
Comment (afc163) [inline components/grid/demo/flex-fill-rest.md]: Flex layout with fixed width column.

File: components/grid/demo/flex-fill-rest.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+order: 7
+title:
+  zh-CN: Flex 填充
```
Comment (afc163) [inline components/grid/demo/flex-fill-rest.md]: Fixed width column

File: components/grid/demo/flex-fill-rest.md (hunk 3 above)
Comment (afc163) [reply components/grid/demo/flex-fill-rest.md]: 没懂。

**Comment by zombieJ** [comment]
Close since https://github.com/ant-design/ant-design/issues/7309#issuecomment-470050192

**Pull request closed without merging**